Jellyfin.Api/ModelBinders/CommaDelimitedArrayModelBinder.cs

```diff
@@ -55,5 +55,36 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
             return Task.CompletedTask;
         }
+
+        private Array GetParsedResult(IReadOnlyList<string> values, Type elementType, TypeConverter converter)
+        {
+            var parsedValues = new object?[values.Count];
+            var convertedCount = 0;
+            for (var i = 0; i < values.Count; i++)
+            {
+                try
+                {
+                    parsedValues[i] = converter.ConvertFromString(values[i]?.Trim());
+                    convertedCount++;
```

Comment:
If parsedValues[i] == null (as per line 83) - then won't typedValues be defined larger than is required, resulting in trailing nulls?

Shouldn't the parsedValues[i] != null be moved from line 83 to here?

Reply:
`parsedValues[i]` is only null when the converter throws an exception. `typedValues` is created at a length of however many values were converted, so it is always the correct length. If it were to be created at the original length, there would be extra `default` values in the resulting array.